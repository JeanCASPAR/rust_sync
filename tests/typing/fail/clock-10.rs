use rust_sync::sync;

sync! {
    #![pass(1)]

    node oui(c1 : bool, c2 : bool) = ()
    where
        a : float = merge c1 {
            true => 3.0 when c1,
            false => 2.0 whennot c2,
        };
}

fn main() {}
