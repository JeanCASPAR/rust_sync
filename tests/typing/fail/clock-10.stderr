error: proc macro panicked
  --> tests/typing/fail/clock-10.rs:3:1
   |
3  | / sync! {
4  | |     #![pass(1)]
5  | |
6  | |     node oui(c1 : bool, c2 : bool) = ()
...  |
11 | |         };
12 | | }
   | |_^
   |
   = help: message: not yet implemented: raise error: the false branch doesn't have the right clock
